packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/BlocksInput/Blocks/Code.tsx

```diff
@@ -1,20 +1,240 @@
 import * as React from 'react';
 
+import { Box, SingleSelect, SingleSelectOption } from '@strapi/design-system';
 import { Code } from '@strapi/icons';
+import { useIntl } from 'react-intl';
+import { Editor, Transforms } from 'slate';
+import { useSelected, type RenderElementProps, useFocused, ReactEditor } from 'slate-react';
 import { styled } from 'styled-components';
 
-import { type BlocksStore } from '../BlocksEditor';
+import { useBlocksEditorContext, type BlocksStore } from '../BlocksEditor';
 import { baseHandleConvert } from '../utils/conversions';
 import { pressEnterTwiceToExit } from '../utils/enterKey';
 import { type Block } from '../utils/types';
 
+const languages: { value: string; label: string }[] = [
```

Comment:
I know we are using it only here but considering its extension what do you think about moving this to a constants file? We do the same, for example, with permissions constants on plugins 🤔